ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+import { I18nContext } from '../../../contexts/i18n';
+import AdvancedGasControlsRow from './advanced-gas-controls-row.component';
+
+export default function AdvancedGasControls() {
+  const t = useContext(I18nContext);
+
+  return (
+    <div className="advanced-gas-controls">
+      <AdvancedGasControlsRow
+        titleText={t('gasLimit')}
+        tooltipText=""
+        titleDetailText=""
+      />
+      <AdvancedGasControlsRow
+        titleText={t('maxPriorityFee')}
+        tooltipText=""
+        titleDetailText={
+          <div>
```

Comment:
```js
<Typography variant={not_a_div} fontWeight={FONT_WEIGHTS.BOLD} />
```

(pseudo code, 😬)

Reply:
With regard to this though, we have some text that's bold and some that's not -- what's our preferred way of doing that?  `<typography BOLD>Text</t> 1 GWEI`?